qa/integration-tests/src/test/java/io/zeebe/broker/it/clustering/topology/TopologyClusterTest.java

```diff
@@ -85,5 +85,6 @@ public void shouldExposeClusterSettings() {
     assertThat(topology.getClusterSize()).isEqualTo(CLUSTERING_RULE.getClusterSize());
     assertThat(topology.getPartitionsCount()).isEqualTo(CLUSTERING_RULE.getPartitionCount());
     assertThat(topology.getReplicationFactor()).isEqualTo(CLUSTERING_RULE.getReplicationFactor());
+    assertThat(topology.getGatewayVersion()).isNotBlank();
```

Comment:
Can't we assert the correct version?

Reply:
It's null when we get it in the test

Reply:
Hm, not very helpful to test I guess :sweat_smile: Is that always or just when running them from the IDE? I think the maven tests should run with the JAR itself, no? :thinking: